src/resolvers-cassandra/Messages/queries.js

```diff
@@ -12,26 +12,33 @@ const trackEvent = require('../../clients/appinsights/AppInsightsClient').trackE
  */
 
 function eventToFeature(row) {
+  const FeatureType = 'MultiPoint';
+
   return {
-    type: row.pipelinekey,
-    coordinates: [],
+    type: FeatureType,
+    coordinates: row.computedfeatures.places.map(place=>[place.centroidlon, place.centroidlat]),
```

Comment:
NB: wouldn't these be placeids from the events table? https://github.com/CatalystCode/project-fortis-pipeline/blob/32e8c1795a787db6ce4ae181e746d4d754f4ae19/ops/storage-ddls/cassandra-setup.cql#L249